installation/resources/crds/application-connector/applications.applicationconnector.crd.yaml

```diff
@@ -1,108 +1,113 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
     "helm.sh/resource-policy": keep
   name: applications.applicationconnector.kyma-project.io
 spec:
   group: applicationconnector.kyma-project.io
-  version: v1alpha1
-  scope: Cluster
-  names:
-    plural: applications
-    singular: application
-    kind: Application
-    shortNames:
-    - app
-  validation:
-    openAPIV3Schema:
-      properties:
-        spec:
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
           properties:
-            compassMetadata:
-              type: object
-              required:
-              - "authentication"
+            spec:
               properties:
-                authentication:
+                compassMetadata:
```

Comment:
The issue was suppose to change resource version not CRD API

Reply:
@werdes72 If CRD needs API changes this should be done in different PR. Additionally CRD update is incompatible which required new CRD version and application refactoring

Reply:
Sorry I was confused by the git diff, there are no API changes

Reply:
Don't worry, me too, until I checked those changes locally.